README.md

```diff
@@ -1,5 +1,8 @@
 Second Screen Community Group Charter
 =======
 
-This repository is used to solicit input from the wider community regarding the [Second Screen Community Group Charter](https://webscreens.github.io/cg-charter/). For historical context, please see the
-[rechartering of the Second Screen Community Group issue](https://github.com/w3c/presentation-api/issues/220).
+This repository is used to hold working drafts of the charter of the Second
```

Comment:
I read "working drafts" as suggesting that the approved version of the charter is to be found elsewhere. I would rather just say "This repository is used to hold the charter of the Second Screen Community Group".

Reply:
Sounds good, done.